.github/scripts/trymerge.py

```diff
@@ -998,23 +1008,25 @@ def find_matching_merge_rule(pr: GitHubPR,
     """Returns merge rule matching to this pr or raises an exception"""
     changed_files = pr.get_changed_files()
     approved_by = set(pr.get_approved_by())
-
-    issue_link = (f"https://github.com/{pr.org}/{pr.project}/issues/new"
-                  f"?labels=module%3A%20ci&template=bug-report.yml")
-    bug_report = (f"Please help file a [bug report]({issue_link}) "
-                  f"to PyTorch DevX Team with the PR number.")
+    issue_link = get_new_issue_link(
+        org=pr.org,
+        project=pr.project,
+        labels=["module: ci"],
+    )
+    reject_reason = (f"No rules find to match PR, "
+                     f"please [report]{issue_link} this issue to DevX team")
 
     try:
-        # Any error here means that the script couldn't load the merge rules for
-        # some reasons, and the error should say so
         rules = read_merge_rules(repo, pr.org, pr.project)
-    except FileNotFoundError as error:
-        raise RuntimeError(str(error))
-    except RuntimeError as error:
-        raise RuntimeError(f"{error}. {bug_report}")
+    except Exception as error:
+        # Any error here means that the script couldn't load the merge rules for
+        # some reasons, and the reject message should say so
+        raise RuntimeError(f"{reject_reason}: {error}")
+
+    if not rules:
+        reject_reason = "Merges are not allowed into repository without a rules."
```

Comment:
"Without a rules" is a bit hard to understand. Can we make the suggested action more explicit?

Perhaps:

```suggestion
        reject_reason = "No matching merge rules found in merge_rules.yaml. Please fix file contents to ensure we have a valid merge rule are defined."
```

Reply:
Perhaps "Rejecting the merge as no rules are defined for the repository."

Also, let's not hardcode file name multiple times in the code, but rather move it into top level variable or something